### ISSUE TITLE
Using zipcelx in ie11

### ISSUE THREAD

**Issue by atelierJVA** [egeriis/zipcelx#41]
I am building my app with Webpack and transpiring with Babel, but it crashes in IE11 (the other browsers work fine)
Any suggestion of how to make it work with IE11 —if possible—?

Thanks in advance

**Comment by egeriis** [comment]
It purposefully doesn't transpile into ES5 🙂 

babel is only used for jest. So it may be some leftovers that babel-loader is still there. Thanks for flagging this.

Did you link your the browser crash to non-ES5 build?

**Comment by atelierJVA** [comment]
Yes, because zipcelx is using features like arrow functions or promises that are not supported in IE11.

To confirm it I tested and app that does not work when using the npm module:
```
import zipcelx from 'zipcelx'
const myData = [[ { value: 1, type: 'number' }, {value: 'Apple',type: 'string'}]];
 function  export() {
    const config = {
    filename: 'my_data_',
    sheet: {
        data: myData
        }
    };
    zipcelx(config);
}
export();
``` 
If instead I build zipcelx from sources with web pack/babel, it works fine
